deisctl/units/deis-controller.service

```diff
@@ -6,8 +6,10 @@ After=deis-store-volume.service
 [Service]
 EnvironmentFile=/etc/environment
 TimeoutStartSec=20m
+ExecStartPre=-/usr/bin/etcdctl mkdir /deis/scheduler >/dev/null 2>&1
```

Comment:
Ok, but if it failed due to etcd errors, don't we want to exit there? This is how we accommodate that elsewhere, but it'd make for a funky one-liner: https://github.com/deis/deis/blob/b3a352d8722219f04daccc5c262c0d711d88c098/controller/bin/boot#L27-L35

Not sure what to do about that.